greengrass_mqtt_subscribe_block.py

```diff
@@ -14,16 +14,28 @@ class GreenGrassMQTTSubscribe(GreenGrassMQTTBase):
 
     def configure(self, context):
         super().configure(context)
-        self.client.subscribe(topic=self.topic(),
-                              QoS=0,
-                              callback=self.handle_message)
+        response = self.client.subscribe(topic=self.topic(),
```

Comment:
This should be in `start` instead of `configure`. A block must not notify signals in `configure`.